src/hooks/useShardsList.ts

```diff
@@ -56,8 +57,7 @@ const useShardsList = <T extends LiveSpecsExtBaseQuery>(specs: T[]) => {
                 .catch((error: string) => {
                     if (
                         error.includes(ErrorFlags.TOKEN_INVALID) ||
-                        error.includes(ErrorFlags.TOKEN_NOT_FOUND) ||
-                        error.includes(ErrorFlags.OPERATION_INVALID)
```

Comment:
Invalid operations should never result in the generation of a new auth gateway URL and token. This line was introduced as a temporary patch while an update to the `data-plane-gateway` was underway and can safely be removed now.